crates/rattler_conda_types/src/repo_data/sharded.rs

```diff
@@ -19,7 +19,7 @@ pub struct ShardedRepodata {
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct ShardedSubdirInfo {
     /// The name of the subdirectory
-    pub subdir: String,
+    pub subdir: Platform,
```

Comment:
Is this important? The reason we use a string instead of a platform is that by using a string we can also support records and shards for platforms that we dont support directly.

Reply:
not really, will revert